### PR TITLE
Document in development doc how to run coverage locally.

### PR DESCRIPTION
**Pull request by hzeller**
Also provide script to go along with it for ease-of-use.

Signed-off-by: Henner Zeller <hzeller@google.com>

### CODE REVIEW

File: .github/bin/generate-coverage-html.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+#
+# Precondition, having run bazel coverage with coverage data collection, e.g.
+#   MODE=coverage .github/bin/build-and-test.sh
+#
+
+PROJECT_ROOT=$(dirname $0)/../../
+OUTPUT_DIR=coverage-html
+
+cd $PROJECT_ROOT
+genhtml -o ${OUTPUT_DIR} bazel-out/_coverage/_coverage_report.dat
```
Comment (fangism) [inline .github/bin/generate-coverage-html.sh:37]: genhtml is a tool from another required package?  if so, include in comment above.

**Review by fangism — APPROVED** [review]
😍

File: .github/bin/generate-coverage-html.sh (hunk 1 above)
Comment (hzeller) [reply .github/bin/generate-coverage-html.sh:37]: It is mentioned above in the readme: `gcov` is the package to install.
That is also the package to install to allow bazel coverage to work in the first place.

Comment (hzeller) [reply .github/bin/generate-coverage-html.sh:37]: mmh, maybe it is lcov, let me check.

Comment (hzeller) [reply .github/bin/generate-coverage-html.sh:37]: It was `lcov`. Fixed.

**Pull request merged**